.serena/project.yml

```
@@ -1,15 +1,25 @@
+
+
```

YAMLlint (1.38.0)

[error] too many blank lines (2 > 0)

(empty-lines)